rpi-convert-stage-5.sh

```diff
@@ -190,13 +189,6 @@ do_install_bootloader() {
     exit 1
   fi
 
-  local gcc_version=$(${bootloader_toolchain}-gcc -dumpversion)
-
-  if [ $(version $gcc_version) -ne $(version $GCC_VERSION) ]; then
```

Comment:
Function version() is used only here, so you could also remove the function definition.

Reply:
done